apps/expo/package.json

```diff
@@ -4,9 +4,9 @@
   "main": "index.ts",
   "scripts": {
     "clean": "rm -rf .expo .turbo node_modules",
-    "dev": "expo start --ios",
-    "dev:android": "expo start --android",
-    "dev:ios": "expo start --ios",
+    "dev": "npx expo start --ios",
+    "dev:android": "npx expo start --android",
+    "dev:ios": "npx expo start --ios",
```

Comment:
Revert these, no need to specify npx here. You only need that when you run it from the terminal

<img width="722" alt="CleanShot 2023-02-23 at 19 59 46@2x" src="https://user-images.githubusercontent.com/51714798/221004492-cdc26fe7-116c-4816-bb9e-b4f117e0a011.png">

Reply:
Thanks for the correction. I will open a new PR with the `npx` commit excluded.